.devcontainer/default/devcontainer.json

```
@@ -0,0 +1,33 @@
+// Reference: https://containers.dev/implementors/json_reference/
+{
+  "name": "SourceKit-LSP",
+  "dockerFile": "Dockerfile",
+
+  // Allow the processes in the container to attach a debugger
+  "capAdd": [ "SYS_PTRACE" ],
+  "securityOpt": [ "seccomp=unconfined" ],
+
+  "mounts": [
+    // Use a named volume for the build products for optimal performance (https://code.visualstudio.com/remote/advancedcontainers/improve-performance?WT.mc_id=javascript-14373-yolasors#_use-a-targeted-named-volume)
```

Do you know where the `WT.mc_id=javascript-14373-yolasors` in the URL is coming from? Removing it seems to work as well

```suggestion
    // Use a named volume for the build products for optimal performance (https://code.visualstudio.com/remote/advancedcontainers/improve-performance#_use-a-targeted-named-volume)
```